reloadButton.js

```diff
@@ -0,0 +1,35 @@
+var href = window.location.href
+var hobostyle = require('hobostyle')
+
+module.exports = function refreshButton(_options) {
```

Comment:
I'm not seeing where this is used?
there is another refreshButton in the mount.js file
also - we should add this to every mounting thing shouldn't we?